task/task-1.py

```diff
@@ -1,9 +1,9 @@
 from utils.reader import read_file
-
-list_data = read_file()
-for data in list_data:
+file_read_csv = read_file("/home/zia/ISDP/Hammad sab/Python/wether_Mar/weather task 2 in csv file/files/f1.csv")
```

Comment:
always 2 new lines space after imports. this mistake should not happen again.

Reply:
ok Inshallah